### PR TITLE
Docs: creating custom machineType

### PR DESCRIPTION
**Pull request by msolters**
Improves documentation surrounding compute instances & compute instance templates needing a custom machineType

### CODE REVIEW

File: website/docs/r/compute_instance.html.markdown [hunk 1]
```diff
@@ -61,10 +61,7 @@ The following arguments are supported:
 * `boot_disk` - (Required) The boot disk for the instance.
     Structure is documented below.
 
-* `machine_type` - (Required) The machine type to create. To create a custom
-    machine type, value should be set as specified
-    [here](https://cloud.google.com/compute/docs/reference/latest/instances#machineType).
-    **Note**: [`allow_stopping_for_update`](#allow_stopping_for_update) must be set to true in order to update this field.
+* `machine_type` - (Required) The machine type to create. To create a custom                                                          machine type, value should be formatted like `custom-4-5120[-ext]` ([GCP docs on machineType](https://cloud.google.com/compute/docs/reference/latest/instances#machineType))
```
Comment (danawillow) [inline website/docs/r/compute_instance.html.markdown]: Looks like a whole bunch of spaces snuck in here, mind fixing that?

Comment (msolters) [reply website/docs/r/compute_instance.html.markdown]: 👍

File: website/docs/r/compute_instance.html.markdown [hunk 2]
```diff
@@ -61,10 +61,7 @@ The following arguments are supported:
 * `boot_disk` - (Required) The boot disk for the instance.
     Structure is documented below.
 
-* `machine_type` - (Required) The machine type to create. To create a custom
-    machine type, value should be set as specified
-    [here](https://cloud.google.com/compute/docs/reference/latest/instances#machineType).
-    **Note**: [`allow_stopping_for_update`](#allow_stopping_for_update) must be set to true in order to update this field.
```
Comment (rosbo) [inline website/docs/r/compute_instance.html.markdown]: This **Note** should also stay there.

Comment (rosbo) [inline website/docs/r/compute_instance.html.markdown]: This **Note** should also stay there.

**Comment by sethvargo** [comment]
FYI I pull these in, rebased, and updated the notes in GH-1574

**Comment by rosbo** [comment]
Closing in favor of #1574

**Pull request closed without merging**